operators/index.adoc

```diff
@@ -8,12 +8,16 @@ toc::[]
 
 include::modules/operators-overview.adoc[leveloffset=+1]
 
-With Operators, you can create applications to monitor the running services in the cluster. Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
+Operators are designed specifically for Kubernetes-native applications to implement and automate common Day 1 operations on a {product-title} cluster, such as installation and configuration. Operators can also automate Day 2 operations, such as autoscaling up and down and creating backups. All these activities are in a piece of software running inside your cluster.
 
-[id="operators-overview-developer-tasks"]
+Operator Lifecycle Manager (OLM) and OperatorHub are default components in {product-title} that help manage add-on Operators. Together they provide the system for discovering, installing, and managing optional Operators available on the cluster, also known as _OLM-based Operators_.
```

Comment:
Yes, so this seems to tie-in with bullet #2. And I have no idea what OperatorHub is here. Sudden mention without explanation. Should OperatorHub be listed in the preceding bullet as well, or is it just an interface for OLM, etc?